oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/IndexImporterSupportBase.java

```diff
@@ -40,15 +40,17 @@ public IndexImporterSupportBase(IndexHelper indexHelper) {
     }
 
     public void importIndex(File importDir) throws IOException, CommitFailedException {
-        IndexImporter importer = new IndexImporter(
-                nodeStore,
-                importDir,
-                createIndexEditorProvider(),
-                createLock(),
-                indexHelper.getStatisticsProvider(),
-                indexHelper.getIndexReporter());
-        addImportProviders(importer);
-        importer.importIndex();
+        try (IndexEditorProvider providers = createIndexEditorProvider()) {
```

Comment:
Looks like an unrelated commit.

Reply:
Yes, I'm piggybacking this, a previous PR was missing a call to `close()` here.